src/main/java/org/spongepowered/api/world/gamerule/DefaultGameRules.java

```diff
@@ -24,25 +24,169 @@
+import org.spongepowered.api.entity.living.monster.Skeleton;
+import org.spongepowered.api.entity.living.monster.Wither;
+import org.spongepowered.api.entity.living.monster.Zombie;
+import org.spongepowered.api.entity.living.monster.ZombiePigman;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.entity.living.player.gamemode.GameModes;
+
 /**
  * An enumeration of all possible GameRule names in vanilla minecraft.
  */
 public class DefaultGameRules {
 
+    /**
+     * Whether {@link CommandBlock}s should notify admins when
+     * they perform commands.
+     *
+     * <p>This is a boolean game rule.</p>
+     */
     public static final String COMMAND_BLOCK_OUTPUT = "commandBlockOutput";
+
+    /**
+     * Whether the server should skip checking player speed when
+     * the player is wearing elytra.
+     *
+     * <p>This is a boolean game rule.</p>
+     */
+    public static final String DISABLE_ELYTRA_MOVEMENT_CHECK = "disableElytraMovementCheck";
+
+    /**
+     * Whether the day-night cycle and moon phases progress.
+     *
+     * <p>This is a boolean game rule.</p>
+     */
     public static final String DO_DAYLIGHT_CYCLE = "doDaylightCycle";
+
+    /**
+     * Whether entities that are not mobs should have drops.
+     *
+     * <p>This is a boolean game rule.</p>
+     */
     public static final String DO_ENTITY_DROPS = "doEntityDrops";
+
+    /**
+     * Whether fire should spread and naturally extinguish.
+     *
+     * <p>This is a boolean game rule.</p>
+     */
     public static final String DO_FIRE_TICK = "doFireTick";
+
+    /**
+     * Whether {@link Agent}s should drop items.
+     *
+     * <p>This is a boolean game rule.</p>
+     */
     public static final String DO_MOB_LOOT = "doMobLoot";
+
+    /**
+     * Whether {@link Agent}s should naturally spawn.
+     *
+     * <p>This is a boolean game rule.</p>
+     */
     public static final String DO_MOB_SPAWNING = "doMobSpawning";
+
+    /**
+     * Whether blocks should have drops.
+     *
+     * <p>This is a boolean game rule.</p>
+     */
     public static final String DO_TILE_DROPS = "doTileDrops";
+
+    /**
+     * Whether {@link Player}s should keep items in their inventory
+     * after death.
+     *
+     * <p>This is a boolean game rule.</p>
+     */
     public static final String KEEP_INVENTORY = "keepInventory";
+
+    /**
+     * Whether to log admin commands to server log.
+     *
+     * <p>This is a boolean game rule.</p>
+     */
     public static final String LOG_ADMIN_COMMANDS = "logAdminCommands";
+
+    /**
+     * Whether creepers, {@link Zombie}, {@link Enderman}, {@link Ghast},
+     * {@link Wither}, {@link EnderDragon}, {@link Rabbit}, {@link Sheep},
+     * and {@link Villager} should be able to change blocks, and whether
+     * {@link Villager} , {@link Zombie}, {@link Skeleton}, and
+     * {@link ZombiePigman} can pick up items.
+     *
```

Comment:
Snow golems and silverfish can also modify blocks, and this also includes every other living entity but only for falling on farmland which will turn it into dirt.